src/EvoSC.Common/Database/DatabaseConfigExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using EvoSC.Common.Config.Models;
+
+namespace EvoSC.Common.Database;
+
+public static class DatabaseConfigExtensions
+{
+    public static string GetConnectionString(this IDatabaseConfig config)
+    {
+        switch (config.Type)
+        {
+            case IDatabaseConfig.DatabaseType.MySql:
+                return $"Server={config.Host};Port={config.Port};Database={config.Name};Uid={config.Username};Pwd={config.Password}";
```

Comment:
Can you elaborate on this? Do you want an additional config option for custom connection strings?

Reply:
There's a few reasons, i.e. defining failover database servers, requiring SSL mode, etc.
See [MySQL Docs](https://dev.mysql.com/doc/connector-net/en/connector-net-connections-string.html#connector-net-connections-open) as reference.

Reply:
Wouldn't it be better to provide these as config options, so the user don't need to look up mysql docs and we can also validate for the users so they don't do something wrong?

Reply:
In theory yes, however since most of those have default values and only need to be set when overridden (like SslMode) I'm not quite sure how we would go about this.

Reply:
Continue discussion in #48